css/src/components/steps.scss

```diff
@@ -0,0 +1,20 @@
+$steps-gap: 7px;
+$steps-size: 8px;
+$steps-size-double: 2 * $steps-size;
+
+.steps {
+	display: flex;
+	gap: $steps-gap;
+
+	.step {
+		width: $steps-size;
+		height: $steps-size;
+		border-radius: $border-radius-rounded;
+		background-color: $secondary;
+
+		&.step-selected {
```

Comment:
What do you think about something like `step-current` instead to indicate current progress? Feel free to resolve if you don't see a difference.